lib/core-integration/src/Test/Integration/Scenario/API/Network.hs

```diff
@@ -104,14 +104,13 @@ spec = do
         let networkParams = getFromResponse id r
         networkParams `shouldBe`
             toApiNetworkParameters (ctx ^. #_networkParameters)
-        let Right zeroPercent = Quantity <$> mkPercentage 0
+        let Right zeroPercent = Quantity <$> mkPercentage 75
```

Comment:
?

Reply:
yeah, why `75`?

Reply:
Because the d-parameter is now set to `0.25` (was 1) to allow both the BFT node and pools to produce blocks. We actually report (1 - d) in the API, hence 75% here. We are 75% decentralized in the integration test.